spec/person_spec.rb

```diff
@@ -0,0 +1,8 @@
+require_relative '../person'
+
+describe Person do
+  it 'should create instance of the person class with 3 parameters' do
+    person = Person.new(age: 18, name: 'Ali')
+    expect(person).to be_an_instance_of Person
+  end
+end
```

Comment:
-  [ ] [REQUIRED] Hello team, great job so far but it seems some methods have not been tested.
Let's test the `add_rental`, `correct_name`, `can_use_services?`, `of_age?` methods for this class as stated in the project requirements.